egs/europarl_st/SRT/prepare.sh

```
@@ -0,0 +1,96 @@
+
+# Paths (modify as needed)
+data_root=./data/europarl_st
+raw_dir=${data_root}/v1.1
+audio_dir=${data_root}/audio
+texts_dir=${data_root}/texts
+norm_dir=${data_root}/normalizer
+manifest_dir=${data_root}/manifests
+fbank_dir=${data_root}/fbank
+bpe_dir=${data_root}/bpe
+
+. shared/parse_options.sh || true
```

Shellcheck (0.11.0)

[info] Not following: shared/parse_options.sh was not specified as input (see shellcheck -x).

(SC1091)